r/tests/testthat/test-RecordBatch.R

```diff
@@ -15,7 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-context("RecordBatch")
+library(dplyr)
```

Comment:
Restore the `context()` function here, and use `dplyr::` before `group_by()` below so you don't need to load dplyr here
```suggestion
context("RecordBatch")
```

Reply:
In PRs on other tests, I think some people have been removing `context` as it doesn't add anything useful to the CI output, so I've just been getting rid of it when I've been working with certain files.  Happy to add back in, but sounds like more discussion needed?

Cool, won't import `dplyr` here.

Reply:
Ah, understood, thanks. I modified these suggestions accordingly.

Reply:
Thank you!